builtin-functions/kphp-full/_functions.txt

```diff
@@ -1584,13 +1584,13 @@ class JsonEncoder {
   static function from_json_impl(string $encoder_tag, string $json, string $class_name) ::: instance<^3>;
 }
 
-class DateTimeZone {
+final class DateTimeZone {
```

Comment:
why? in php these are not `final` classes. We support inheritance from classes in the standard library

Reply:
Fixed. Kept them non-`final`. Also, fixed the declarations of such classes so they can really be extended in user code